static/app/views/userFeedback/index.tsx

```diff
@@ -148,14 +147,21 @@ class OrganizationUserFeedback extends AsyncView<Props, State> {
                   <EnvironmentPageFilter />
                   <DatePageFilter alignDropdown="right" />
                 </PageFilterBar>
-                <ButtonBar active={!Array.isArray(status) ? status || '' : ''} merged>
-                  <Button barId="unresolved" to={{pathname, query: unresolvedQuery}}>
+                <SegmentedControl
+                  aria-label="Issue Status"
```

Comment:
Does this need a translation?

Reply:
oops yes! https://github.com/getsentry/sentry/pull/44073/commits/0193b0f7a52131b52ae137d7b7286ffd4389f9d6